examples/ui/ui.rs

```diff
@@ -309,6 +316,31 @@ struct ScrollingList {
     position: f32,
 }
 
+fn change_ui_camera(
```

Comment:
IMO we should have this in its own example. I don't like how big this example is to begin with; let's not make it worse.